api/v1alpha1/authorization_types.go

```diff
@@ -0,0 +1,55 @@
+// Copyright Envoy Gateway Authors
+// SPDX-License-Identifier: Apache-2.0
+// The full text of the Apache license is available in the LICENSE file at
+// the root of the repo.
+
+package v1alpha1
+
+// Authorization defines the authorization configuration.
+type Authorization struct {
+	// Rules contains all the authorization rules.
+	// If rules contains at least one Allow rule and none of them
+	// matches the action for the request is deny.
+	// If rules contains at least one Deny rule and none of them
+	// matches the action for the request is allow.
+	// If rules contains Allow and Deny,
+	// and the request matches both, `Deny` takes precedence.
+	//
+	// +kubebuilder:validation:MinItems=1
+	Rules []Rule `json:"rules,omitempty"`
+}
+
+// Rule defines the single authorization rule.
+type Rule struct {
+	// ClientSelectors contains the client selector configuration.
+	// All selectors are AND'd together and only if all selector are valid
+	// the Action is performed.
+	//
+	// +kubebuilder:validation:MinItems=1
+	ClientSelectors []ClientSelector `json:"clientSelector,omitempty"`
+
+	// Action defines the action to be taken if the rule matches.
+	Action RuleActionType `json:"action"`
+}
+
+// ClientSelector contains the client selector configuration.
+type ClientSelector struct {
+	// ClientCIDRs is a list of CIDRs.
```

Comment:
This condition is met if any of the values in the list of CIRDs matches.

Reply:
trying to make sure we add a statement that highlights that if there are a list of CIDRs, and 1 of them match, then the `action` is executed